src/java/fr/paris/lutece/portal/resources/system_messages.properties

```diff
@@ -68,6 +68,7 @@ manage_caches.actionDisable=Turn off
 manage_caches.actionViewKeys=View config and keys
 manage_caches.buttonFlush=Flush all
 manage_caches.buttonViewKeys=View config and keys
+manage_caches.titleReloadLogsConfiguration=Log configuration loading
```

Comment:
Wouldn't that be "reloading" ?

Reply:
I chose the same expression as the other "reloading" button which says "Properties Loading". I don't have a strong opinion